### PR TITLE
Change absolute links to relative links in the HTML-CSS README.md

### PR DESCRIPTION
**Pull request by ChrisOwen101**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,12 +13,12 @@ The command line (cli, shell) is the interface between you (the user) and the op
...
-| 2. | [CLI](https://github.com/HackYourFuture/CommandLine/blob/master/Lecture-1.md), Responsive design, media queries, developer tools | [Week 2 Reading](https://github.com/HackYourFuture/HTML-CSS/tree/master/Week2/README.md) | [Homework week 2](https://github.com/HackYourFuture/HTML-CSS/tree/master/Week2/MAKEME.md) ||[Review](https://github.com/HackYourFuture/HTML-CSS/tree/master/Week2/REVIEW.md)|
-| 3. | [CLI](https://github.com/HackYourFuture/CommandLine/blob/master/Lecture-2.md), Recap, useful resources on the web| [JavaScript Intro](https://github.com/HackYourFuture/JavaScript/tree/laurens_thomas/Week0) | [Homework week 3](https://github.com/HackYourFuture/HTML-CSS/tree/master/Week3/MAKEME.md) ||[Review](https://github.com/HackYourFuture/HTML-CSS/tree/master/Week3/REVIEW.md)|
+| Week | Topic                                    | Read                                     | Homework                                 | Review                                   |
+| ---- | ---------------------------------------- | ---------------------------------------- | ---------------------------------------- | ---------------------------------------- |
```
Comment (mkruijt) [inline README.md:17]: can you fix the markup? This looks really messy. Like this:

| Week | Topic | Read | Homework | Review |
| ----- | ------ | ----- |------------|--------|

Comment (mkruijt) [reply README.md:17]: ah I mean: | ---- | ----- | ---- |----------|------| (get out all the spaces you have now)

**Review by mkruijt — CHANGES_REQUESTED** [review]
Hi @ChrisOwen101 I totally missed this, can you ask me for a review if the next time?

**Comment by ChrisOwen101** [comment]
Ah, I thought that putting you as an "Assignee" would notify you. I'll tag you next time :)

File: README.md (hunk 1 above)
Comment (ChrisOwen101) [reply README.md:17]: I think this is my markdown editor that did this automatically, very annoying

Comment (mkruijt) [reply README.md:17]: hm ok I will fix it myself :)

**Pull request closed without merging**